### DISCUSSION TITLE
How do I use Quarto as replacement for JSDocs to create documentation for my JavaScript Projects?

### DISCUSSION THREAD

**Discussion by BatoolMM** [quarto-dev/quarto-cli#8163] [Q&A]
### Description

We are creating MOOCs to teach R, open Science, reproducibility in Arabic and English - it's all (WIP) in [this repository](https://github.com/Open-Science-Community-Saudi-Arabia/MOOCs). We have used JSDocs to create documentation for API/front-end but we would like to replace it with Quarto? Is there any pointer that you can direct us to - especially if someone used it to create documentation for API?

I am also looping in @sandygudie as the co-developer of the MOOCs!

**Comment by mcanouil** [comment]
There are no guidelines, so you'll have to look at Quarto document for what you need and see how it can work for your use case.

>  create documentation for API/front-end

This is very general making very difficult to answer as nearly everything falls in those "API/front-end".


FYI, for Python module documentation, there is `quartodoc` which help going from Python docstring to Quarto website: https://machow.github.io/quartodoc/get-started/overview.html
